src/components/ui/InfoTooltip/InfoTooltip.tsx

```diff
@@ -0,0 +1,11 @@
+import { FC } from 'react';
+
+import style from './InfoTooltip.module.scss';
+
+interface InfoTooltipProps {
+  info: string | number;
```

Comment:
Is it a component for amount of products ?
Then why string or number ?

Reply:
It is a ui component which will be reused on single product page. it contains non number values such as ' - 30%' etc.
However, number type does not make sense here so I left string only and updated relevant components.